spec/route/account/_id/index.spec.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+
+var accountManagerMock, promiseMock, sessionManagerMock;
+
+jasmine.routeTester("/account/_id/", (container) => {
+    accountManagerMock = require("../../../mock/account-manager-mock")();
+    promiseMock = require("../../../mock/promise-mock")();
+    sessionManagerMock = require("../../../mock/session-manager-mock")();
+    container.register("accountManager", accountManagerMock);
+    container.register("sessionManager", sessionManagerMock);
+}, (routeTester) => {
+    beforeEach(() => {
+        routeTester.req.params.id = "account-id";
+    });
+    it("exports GET and a name", () => {
+        expect(Object.keys(routeTester.exports).sort()).toEqual([
+            "get",
+            "name"
+        ]);
+    });
+
+    describe("GET", () => {
+        describe("success", () => {
+            beforeEach((done) => {
+                routeTester.req.cookies.login = "asdf";
+                routeTester.get().then(done);
+            });
+            it("refreshes the login cookie", () => {
+                expect(routeTester.res.setCookie).toHaveBeenCalledWith("login", "asdf", {
+                    httpOnly: true,
+                    maxAge: jasmine.any(Number),
+                    path: "/account/",
+                    secure: true
+                });
+            });
+            it("returns the account record", () => {
+                expect(routeTester.res.send).toHaveBeenCalledWith(200, {
+                    challengeHashConfig: "record-challenge-hash-config",
+                    email: "record-email",
+                    passwordHashConfig: "record-password-hash-config"
+                });
+            });
+            it("creates the right links", () => {
+                expect(routeTester.res.linkObjects).toEqual([
+                    {
+                        href: "rendered route: account-logout, id:\"account-id\"",
+                        profile: "/schema/account/logout.json",
+                        rel: "service",
+                        title: "account-logout"
+                    }
+                ]);
+            });
+        });
+
+        // This can't happen because the cookie value is used for login.
+        // However, assuming it *could* happen, let's make sure that
+        // we do not permit this sort of behavior.  Because the managers
+        // are mocked to allow any value, we can simulate this bizarre
+        // behavior quite easily.
+        describe("no login cookie to refresh", () => {
+            it("errors when refreshing the login cookie", (done) => {
+                routeTester.get().then(() => {
+                    jasmine.fail();
+                }, (err) => {
+                    expect(err.toString()).toContain("No login cookie to refresh");
+                }).then(done, done);
```

Comment:
Thought we got rid of this and you can just return the `routeTester.get()` promise.